Changed 10mm ammo and Ringneck spread

dahlonl opened the pull request:
## About The Pull Request
Increased 10mm ammo damage
Increased Ringneck spread

## Changelog

:cl:

balance: 10mm ammo and Ringneck spread.

/:cl:

dahlonl committed:
Changed 10mm ammo and Ringneck spread

TDHooligan commented:
initial changes look good. 
look at /obj/item/gun/ballistic/automatic to see the firerate options, i'd cursorily say the 10mm needs a 15% firerate debuff to make sure that .45 hits certain breakpoints faster.

will do the proper math later.

dahlonl committed:
Rebalance of other 10mm weapons

TDHooligan commented:
you can add the fire_delay var defined inside /obj/item/gun to any of the guns (as they are also /obj/item/gun)

since they're not populated already, you'll have to add the definition, like how the other guns do. for automatics the fire_delay is 0.4 SECONDS, so add 0.5 SECONDS for the SMG, and that should be it

dahlonl committed:
attempt at balancing 10mm weapons more
reverted some recoil on the SMG but shoots slower
changed mag cap for 10mm weapons

TDHooligan commented:
looks good to me!
